lib/quickbooks/model/reports.rb

```diff
@@ -1,17 +0,0 @@
-module Quickbooks
-  module Model
-    class Reports < BaseModel
-      XML_COLLECTION_NODE = "Reports"
-      XML_NODE = "Reports"
-      REST_RESOURCE = 'reports'
-
-      REPORT_TYPES = ['BalanceSheet', 'ProfitandLoss', 'ProfitAndLossDetail', 'TrialBalance', 'CashFlow', 'InventoryValuationSummary', 'CustomerSales', 'ItemSales', 'DepartmentSales', 'ClassSales', 'CustomerIncome', 'CustomerBalance', 'CustomerBalanceDetail', 'AgedReceivables', 'AgedReceivableDetail', 'VendorBalance', 'VendorBalanceDetail', 'AgedPayables', 'AgedPayableDetail', 'VendorExpenses', 'GeneralLedgerDetail']
```

Comment:
These weren't used and were out of date w/ all the supported reports now, so I removed them